README.md

```diff
@@ -12,7 +12,8 @@ Node.js module.
   - [This is an alpha 1.x version](#alpha)
   - [Not accepting 0.x fixes anymore](#0.x-fixes)
 - [About Globalize](#about)
-  - [Why globalization?](#why)
+  - [Why globalization?](#why_globalization)
```

Comment:
Would be good to add something along the lines of "Even if you're only dealing with the English locale, you are still likely to need globalization, to display something better than "Edited 1 minutes ago"."